### PR TITLE
Clean up CPM experiment

### PR DESCRIPTION
**Pull request by muodov**
Task/Issue URL: https://app.asana.com/0/1206594217714678/1207420864578039/f
Tech Design URL:
CC:

**Description**:
This PR cleans up CPM filterlist experiment and enables filterlist for all users (still controlled by the remote config).

**Steps to test this PR**:
1. make sure you're using remote config that enables "autoconsent -> filterlist" subfeature
2. go to https://skyscanner.nl, ensure that the cookie pop-up is not visible.

**Definition of Done**:

* [ ] Does this PR satisfy our [Definition of Done](https://app.asana.com/0/1202500774821704/1207634633537039/f)?

---
###### Internal references:
[Pull Request Review Checklist](https://app.asana.com/0/1202500774821704/1203764234894239/f)
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
[Pull Request Documentation](https://app.asana.com/0/1202500774821704/1204012835277482/f)

### CODE REVIEW

File: .github/workflows/pr.yml [hunk 1]
```diff
@@ -386,11 +386,20 @@ jobs:
   verify-autoconsent-bundle:
     name: 'Verify autoconsent bundle'
     runs-on: ubuntu-latest
-
     steps:
-      - name: Skip
+      - uses: actions/checkout@v4
```
Comment (federicocappelli) [inline .github/workflows/pr.yml:390]: Are these changes intentional? Where are they coming from? Should revert?

**Review by federicocappelli — CHANGES_REQUESTED** [review]

File: .github/workflows/pr.yml (hunk 1 above)
Comment (muodov) [reply .github/workflows/pr.yml:390]: yep, it's basically reverting the earlier change from [here](https://github.com/duckduckgo/macos-browser/pull/3086#discussion_r1713969651)

**Review by federicocappelli — APPROVED** [review]
LGTM except the BSK reference
